install.python3.py

```diff
@@ -40,7 +40,16 @@ def getIP():
     s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
     s.connect(("8.8.8.8", 80))
     return s.getsockname()[0]
-
+#def getIP():
```

Comment:
Use the original python3 ,,,, rather than updated.
```suggestion
#def getIP():
```